Defs/RuleDefs/BrewingRules.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Defs>
+  <RuleDef>
+    <defName>AddMustToFermentingBarrels</defName>
```

Comment:
It turns out this doesn't seem to be used? I think it is for filling random barrels when spawning a base or something, as it adds random amount of wort (or must for us) and progress to the barrels. 

I'm considering taking this out, or at least seeing if it works without it.